acbuild/acbuild.go

```diff
@@ -239,3 +262,30 @@ func stdout(format string, a ...interface{}) {
 	out := fmt.Sprintf(format, a...)
 	fmt.Fprintln(os.Stdout, strings.TrimSuffix(out, "\n"))
 }
+
+func addACBuildAnnotation(a *lib.ACBuild, cmd *cobra.Command, args []string) error {
+	if a == nil {
+		a = newACBuild()
+	}
+
+	man, err := util.GetManifest(a.CurrentACIPath)
+	if err != nil {
+		return err
+	}
+
+	var acbuildCount int
+	for _, ann := range man.Annotations {
+		var tmpCount int
+		n, _ := fmt.Sscanf(string(ann.Name), "acbuild-command-%d", &tmpCount)
+		if n == 1 && tmpCount > acbuildCount {
+			acbuildCount = tmpCount
+		}
+	}
+
+	command := cmd.Name()
+	for _, a := range args {
+		command += fmt.Sprintf(" %q", a)
+	}
+
+	return a.AddAnnotation(fmt.Sprintf("acbuild-command-%d", acbuildCount+1), command)
```

Comment:
These should actually be namespaces more like... appc.io/acbuild/command or something